bundler.d/test.rb

```diff
@@ -16,4 +14,16 @@
     gem 'rubocop-checkstyle_formatter', '~> 0.2'
     gem 'webmock'
   end
+
+  if RUBY_VERSION < '2.2.2'
+    gem 'rack-test', '~> 0.7.0'
+  else
+    gem 'rack-test'
+  end
+
+  if RUBY_VERSION < '2.2'
```

Comment:
This duplicates the version spec in development.rb; this one can be removed, no?

Reply:
rdoc was part of both files before this change also, I guess when one does exclude one of the two bundler groups, the problem still could happen when it's not in both.